apps/base/utils.py

```diff
@@ -18,20 +17,17 @@ def paginated_queryset(queryset, request):
     return (paginator, result_page)
 
 
-@deconstructible
```

Comment:
@taranjeet I actually misunderstood the use of `deconstructible`. Basically, it is used to add a `deconstruct()` method to a class . The `deconstruct()` method is used to help perform model migrations that aren't handled by the system. Like Suppose if we have a `UUID Field` for a model instance, then django can't serialize that field and we will need a `deconstruct()` method. 
Please refer https://stackoverflow.com/a/35928526 and https://docs.djangoproject.com/en/1.10/topics/migrations/#adding-a-deconstruct-method these links.
So, I think we can remove `@deconstructible` from here. What do you say?